10.1/test/run

```diff
@@ -65,15 +65,21 @@ function test_connection() {
   local max_attempts=20
   local sleep_time=2
   local i
+  echo -n "    Trying to connect..."
   for i in $(seq $max_attempts); do
-    echo "    Trying to connect..."
-    if mysql_cmd "$ip" "$login" "$password" <<< 'SELECT 1;'; then
+    echo -n "."
+    if mysql_cmd "$ip" "$login" "$password" &>/dev/null <<< 'SELECT 1;'; then
+      echo " OK"
       echo "  Success!"
       return 0
     fi
     sleep $sleep_time
   done
-  echo "  Giving up: Failed to connect. Logs:"
+  echo " FAIL"
+  echo "  Giving up: Failed to connect."
+  echo -n "Trying to ping the IP ${ip}: "
+  ping -c 1 ${ip} &>/dev/null && echo "OK" || echo "FAIL"
```

Comment:
Are there systems without ping? :) Well, to be honest, what I really wanted to do is checking whether the container is running, so we should probably do it using `docker inspect...`